saleor/product/models.py

```diff
@@ -629,6 +638,8 @@ class ProductVariantChannelListing(models.Model):
 
     preorder_quantity_threshold = models.IntegerField(blank=True, null=True)
 
+    objects = ProductVariantChannelListingQuerySet.as_manager()
```

Comment:
You can try to rewrite this definition to:
```suggestion
    objects = models.Manager.from_queryset(ProductVariantChannelListingQuerySet)()
```
This is a workaround for mypy to recognize custom queryset method, but in some cases, it doesn't help; I haven't figured out yet when exactly :/

Reply:
Thanks ;) I've burned out the Python environment and I've set it up from scratch, now mypy works correctly on my local machine.
And your solution fixed that issue, thanks 🙇